pkg/apis/autoscaling/v2beta1/conversion.go

```diff
@@ -289,3 +294,46 @@ func Convert_v2beta1_PodsMetricStatus_To_autoscaling_PodsMetricStatus(in *autosc
 	}
 	return nil
 }
+
+func Convert_autoscaling_HorizontalPodAutoscaler_To_v2beta1_HorizontalPodAutoscaler(in *autoscaling.HorizontalPodAutoscaler, out *autoscalingv2beta1.HorizontalPodAutoscaler, s conversion.Scope) error {
+	if err := autoConvert_autoscaling_HorizontalPodAutoscaler_To_v2beta1_HorizontalPodAutoscaler(in, out, s); err != nil {
+		return err
+	}
+	if in.Spec.Behavior != nil {
+		old := out.Annotations
+		out.Annotations = make(map[string]string, len(old)+1)
+		for k, v := range old {
+			out.Annotations[k] = v
+		}
+	}
+
+	if in.Spec.Behavior != nil {
```

Comment:
These 2 blocks are both testing for nil - why 2 blocks?

Reply:
Fixed: https://github.com/kubernetes/kubernetes/pull/74525/commits/877d2ffeccb6d8fd1e47a12eab02de44b8bac07b